theme/block-template-parts/tab-bar.html

```diff
@@ -0,0 +1,23 @@
+<!-- wp:group {"className":"tab-bar__wrap"} -->
+<div class="wp-block-group tab-bar__wrap">
+	<!-- wp:group {"className":"mdc-tab-bar tab-bar"} -->
+	<div class="wp-block-group mdc-tab-bar tab-bar">
+		<!-- wp:group {"className":"mdc-tab-scroller"} -->
+		<div class="wp-block-group mdc-tab-scroller">
+			<!-- wp:group {"className":"mdc-tab-scroller__scroll-area"} -->
+			<div class="wp-block-group mdc-tab-scroller__scroll-area">
+				<!-- wp:group {"className":"mdc-tab-scroller__scroll-content"} -->
+				<div class="wp-block-group mdc-tab-scroller__scroll-content">
+					<!-- wp:navigation-area -->
+						<!-- wp:navigation {"navigationMenuId":654,"showSubmenuIcon":false,"overlayMenu":"never","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"center"}} /-->
```

Comment:
Should this be removed? `"navigationMenuId":654`

How do we provide primary menu as part of block template part?